examples/pimlico/src/account.js

```diff
@@ -5,7 +5,7 @@ import { http, createPublicClient } from 'viem'
 import config from '../../../config.js';
 
 const publicClient = createPublicClient({
-    transport: http(config.rpc_url),
```

Comment:
174.251.164.100

Reply:
Oh I know. It's gonna begin to understand I might be going for a year. I gotta whole Bitcoin. That was $26000 and some so if I wanted to get it out $17500. Want know? I was going to buy nails so from buy nails and bought $17500 worth of your thinking coins at 63 cent a share.  That's the money. I'm that I'm missing right there and I'm won't catch up when in 2 projects. The contact was remind me for the call in an hour to have a good stuff and it's in the candidate program. The guy asked me who I knew. I want to know anybody in the place so he said well. I flagged your account. I made it anonymous and you can't see your money and nobody else can either. However, I earned 565 coins and I'll say the world and run.  And I can't get into my email accoand now, the money's worth 700 and 700 million dollars. So I'd like to have my money. I'm no f****** scary and lying scam nobody. I don't know enough about this. In August 2022. It's when I started know.Ing anything about Bitcoin.
Show the guy that is, a scammer.Is the developer I have a bunch of s***I have everything that he's ever done to me.And PDF files or downloads.And I don't even know how to get it to you

Reply:
![Screenshot_20240512_084536_Microsoft 365 (Office).jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/e116e22b-e29c-4ea7-8bee-3e63faf9d3ea)

Reply:
![IMG-20240511-WA0125.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/52146c73-4852-4846-8c05-d7c1b13f4510)

Reply:
![history.xlsx](https://github.com/coinbase/paymaster-bundler-examples/files/15287120/history.xlsx)

Reply:
https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/0c061264-62e0-4fbc-b034-90d8c6c3e168

Reply:
![IMG-20240511-WA0133.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/1810fddb-7c5d-4304-ab57-ed608603b2a0)

![IMG-20240511-WA0125.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/56db4ea3-613f-4592-b81c-6e61bb9cd6bf)

![IMG-20240511-WA0123.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/eea03f13-b624-4f36-9321-2e76deaddc39)

![IMG-20240511-WA0118.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/760b7343-79cd-47d6-a572-ae87bcb86893)

![IMG-20240511-WA0128.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/a2b22469-5bb5-457f-9d75-56163e49c537)

Reply:
![world-alert@3x.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/eb7d808d-d4ff-48c1-a80c-8b034e7a08af)

<!-- Error while uploading signature.asc -->![USD","tokenSymb.txt](https://github.com/coinbase/paymaster-bundler-examples/files/15287137/USD.tokenSymb.txt)

![SD","tokenSymbo.txt](https://github.com/coinbase/paymaster-bundler-examples/files/15287138/SD.tokenSymbo.txt)

![Cash_App_October_2022_Statement_015eca0bd330e3ef27949242b8b3a550628e7a263ca816652af89d0b48c4b1e1b2a1821f6e539953bbec3417a0f33be544e30fc3c0afc0b73e7171fd0aad870519f7.pdf](https://github.com/coinbase/paymaster-bundler-examples/files/15287139/Cash_App_October_2022_Statement_015eca0bd330e3ef27949242b8b3a550628e7a263ca816652af89d0b48c4b1e1b2a1821f6e539953bbec3417a0f33be544e30fc3c0afc0b73e7171fd0aad870519f7.pdf)

![USD","tokenSymb.txt](https://github.com/coinbase/paymaster-bundler-examples/files/15287136/USD.tokenSymb.txt)

Reply:
![Screenshot_20240328_160506.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/ba10b909-4d92-494b-97c1-475c429044c4)

![Screenshot_20240408_200210_Chrome Beta.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/91acd70c-c725-4da7-86a6-a437aab6745f)

![Screenshot_20240408_194748_Chrome Beta.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/93df1e4c-431c-4037-a815-bcb5c9d54039)

![Screenshot_20240408_200212_Chrome Beta.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/4a4f6dee-5825-48bc-95b7-eefc7b7cb212)

![Screenshot_20240408_200220_Chrome Beta.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/8a34a70a-6ad9-480f-963d-9286afa8152f)

<!-- Error while uploading Screenshot_20240414_124315_All PDF Reader.jpg -->![Screenshot_20240319_073207_Bing.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/7b0d2e56-95ee-4c16-b723-30204c9c2a47)

![Screenshot_20240319_073225_Bing.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/96eb3132-5d83-4bca-87d3-3d3d2869beb4)

![Screenshot_20240408_194802_Chrome Beta.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/c7ab62be-f0e9-4b8d-8c5a-864ebe42a8bd)

![Screenshot_20240319_073154_Bing.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/13d57dd7-de3a-4958-884d-de4bc86e1d56)

![Screenshot_20240319_073203_Bing.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/3a680fb1-0e1d-4c99-98c1-7c72529ce793)

![Screenshot_20240319_071858_Bing.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/006b8693-1818-4e00-a47e-4d9b546de8ab)

![Screenshot_20240319_071822_Bing.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/659e6693-e6e6-4200-a385-e2ec9beb253e)

![Screenshot_20240319_073134_Bing.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/f5525243-c850-44fa-826c-2e0338b79bcf)

![Screenshot_20240319_071834_Bing.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/eca13bb4-7b8e-4524-b536-fd417e1995f6)

![Screenshot_20240311_181836.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/dc213741-2e55-498b-a053-e0b3811a55ce)

![Screenshot_20240227_231625_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/c5491e76-beb6-4fb4-ba36-45d7ea2a2250)

![Screenshot_20240227_231607_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/a954c7c8-0e8f-4006-b196-335a3395b50a)

![Screenshot_20240227_231633_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/cf739f37-e92b-48c9-82cb-00ddc9ec9117)

![Screenshot_20240227_213854_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/4c7bef39-06dd-46bd-b85c-c47bcc0d0ff2)

![Screenshot_20240227_213832_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/7be0be66-dec2-44e6-94d2-fd91845df3ba)

![Screenshot_20240227_213845_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/717eb009-09f0-420e-b2ca-a88f267df489)

![Screenshot_20240227_213812_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/1192c780-f8b2-409b-adce-312d3780aeea)

![Screenshot_20240227_152211_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/036ec0bb-7cb3-41c8-8319-bcc2c609ecb4)

![Screenshot_20240227_152133_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/15130067-6509-434d-9d9c-ae408f86b5e3)

![Screenshot_20240227_152159_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/6680e32d-4195-4a7b-b30f-892108157b01)

![Screenshot_20240227_152126_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/2dbe429e-0e24-466b-be25-4ef349297c96)

![Screenshot_20240227_152146_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/ccdc176d-e632-4ccb-b9da-134dea8da92b)

![Screenshot_20240227_152120_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/9c8cee2a-fb2a-4d7f-98b2-4955e0dab9b3)

![Screenshot_20240227_152109_Chrome.jpg](https://github.![Screenshot_20240227_152102_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/bf05dfcb-66c5-4dab-a980-ccbfa9ccdce2)

/886b16ef-157a-4588-a99d-f603ea5e5699)

![Uploading Screenshot_20240227_152102_Chrome.jpg …]()

![Screenshot_20240222_150407_Drive.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/521ed708-9932-4b57-bd34-ae25a11ae0e2)

![Screenshot_20240219_161522_CoinStats.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/6c60ff35-770c-46a4-9dcc-fd7b69676bf5)

![Screenshot_20240219_161517_CoinStats.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/c4dc3daf-c6cd-4074-ae6b-ced8a8c9ae01)

![Screenshot_20240219_161508_CoinStats.jpg](https://github.![Screenshot_20240203_031419_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/9a6c94e3-3d37-467d-aea4-3a6f92b05353)

/f54d783b-6f6a-4abf-a833-de38239ea7db)

![Uploading Screenshot_20240203_031419_Chrome.jpg …]()

![Uploading Scre![Screenshot_20240203_031334_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/5fa0b6c8-8254-4bcb-beab-7335d3dd4d64)

enshot_20240203_031408_Chrome.jpg …]()

![Uploading Screenshot_20240203_031334_Chrome.jpg …]()

![Uploading Scre![Screenshot_20240203_031303_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/eb74e89b-cc79-4641-8ab7-08a1c23f7b36)

enshot_20240203_031315_Chrome.jpg …]()

![Uploading Screenshot_20240203_031303_Chrome.jpg …]()

![Screenshot_20240203_031249_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/47699e8b-faea-477e-8d35-ba33f7fa2e98)

![Screenshot_20240203_031257_Chrome.jpg](https://github.![Screenshot_20240203_031239_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/e03e8e03-33ab-453d-a4fb-211f2843eda1)

/57a2d3d1-e0fc-45f0-bbdc-3796779d0124)

![Uploading Screenshot_20240203_031239_Chrome.jpg …]()

![Screenshot_20240203_031220_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/1fd067c3-8c4f-47ca-9624-49e12a793256)

![Screenshot_20240203_031210_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/7a417285-965f-45db-9c9b-3b437e5cbce3)

![Screenshot_20240203_031228_Chrome.jpg](https://github.com/coinbase/pay![Screenshot_20240203_031155_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/2ea353d6-56c3-4e70-87da![Screenshot_20240203_031201_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/4172fd7d-c74e-4eb0-953d-837b214c13b0)

enshot_20240203_031149_Chrome.jpg …]()

![Uploading Screenshot_20240203_031201_Chrome.jpg …]()

![Uploading Screenshot_20240203_031155_Chrome.jpg …]()

![Screenshot_20240203_030340_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/25214490-e032-4e8f-830a-b54146830bab)

![Screenshot_20240203_031135_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/9faccd04-f671-4c3a-b7a5-f728ec14bd8e)

![Uploading Scre![Screenshot_20240203_030321_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/cdb14cdd-e51a-41d7-b0eb-db9504399fd9)

40203_030334_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/5338c6d1-4aea-4e0c-9b73-3da7d2eca060)

![Uploading Screenshot_20240203_030321_Chrome.jpg …]()

![Screenshot_20240203_030330_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/dda6b1e8-9720-41a2-92f3-da5424f0f162)

![Screenshot_20240203_030258_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/473b697e-5277-48dd-9e31-ae1cfa036166)

![Screenshot_20240203_030304_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/a7c44344-3811-44f5-9527-9cad947ad379)

![Screenshot_20240203_030253_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/2b17ab11-529e-4c0e-bef2-91212ae9b6d1)

![Screenshot_20240203_030243_Chrome.jpg](https://github.![Screenshot_20240203_030229_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/eca4d54d-0bf6-44d1-8aa4-2c09160a1ded)

/e31f5ccc-d3d0-43f6-a918-3fa964c6151d)

![Uploading Screenshot_20240203_030229_Chrome.jpg …]()

![Screenshot_20240203_030236_Chrome.jpg](https://github.![Screenshot_20240203_030216_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/84c7af81-359c-4adc-8f7f-fea7a0a534fe)

/8154a5d2-91b8-4820-82b6-c27ac047056d)

![Uploading Screenshot_20240203_030216_Chrome.jpg …]()

![Screenshot_20240203_030223_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/c870925c-d595-4fdd-9ae4-336e26616683)

![Uploading Scre![Screenshot_20240203_030210_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/328a7f20-c9f0-4132-8c44-1ed69a0f7eb3)

enshot_20240203_030154_Chrome.jpg …]()

![Uploading Screenshot_20240203_030210_Chrome.jpg …]()

![Screenshot_20240203_030144_Chrome.jpg](https://github.![Screenshot_20240203_030126_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/84f7230d-6e70-4731-b4a7-6a303582cadf)

/10b40927-0946-4e32-850a-ae1b856574c1)

![Uploading Screenshot_20240203_030126_Chrome.jpg …]()

![Screenshot_20240203_030137_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/666ae5dc-17a8-4116-b188-55205b4f34b4)

![Screenshot_20240203_030130_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/b4fc8870-74dc-4249-a3df-7fb4402b1a73)

![Screenshot_20240203_030116_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/bba2858b-a790-415a-9817-1d0a67f7ba10)

![Screenshot_20240203_030103_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/4ffd8db2-7fed-40f7-9928-7f100a4795b3)

![Screenshot_20240203_030111_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/69c467bf-ebd4-4b45-b21c-35ba67515045)

![Screenshot_20240203_030055_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/ecb2461f-df66-40c8-9e15-c17bc85068a8)

![Screenshot_20240203_030050_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/92fc26db-c66b-46f2-951b-271cc5a75d76)

![Screenshot_20240203_030043_Chrome.jpg](https://github.![Screenshot_20240203_030036_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/b46038f0-3394-4326-87a7-ad62a18f8c9e)

/fe971dfe-8d14-4f59-af42-3dccf47f07fe)

![Uploading Screenshot_20240203_030036_Chrome.jpg …]()

![Screenshot_20240203_030030_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/4eddec26-68dd-420d-a360-e0b7ee17825b)

![Screenshot_20240203_030004_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/93cb2628-1cfa-4939-bf79-c8d308116937)

![Screenshot_20240203_025950_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/a0727244-1c16-456f-9e7e-3e54737a5ccf)

![Screenshot_20240203_030021_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/834c2813-6396-4fff-b12f-def91f23137a)

![Uploading Scre![Screenshot_20240203_025930_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/46711feb-9254-403b-99a7-75a3f947dc35)

enshot_20240203_025937_Chrome.jpg …]()

![Uploading Screenshot_20240203_025930_Chrome.jpg …]()

![Uploading Scre![Screenshot_20240203_025848_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/a6a2d277-1974-4f62-86ed-e8f3ae0bb7cf)

enshot_20240203_025923_Chrome.jpg …]()

![Uploading Screenshot_20240203_025848_Chrome.jpg …]()

![Screenshot_20240203_025819_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/4f3a0c90-c136-493d-b698-c14c2d09c79e)

![Screenshot_20240203_025858_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/b290f33a-5215-4c3f-b7b6-83f82ad94425)

![Screenshot_20240203_025828_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/90836531-fc09-46fa-80bd-e6edd8f8ac8c)

![Screenshot_20240203_025832_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/b513d481-ceb8-4d03-bbac-fc2d7c09dd73)

![Screenshot_20240203_020556_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/08dc7279-543e-47e2-a3c9-2a8a8ac30e99)

![Screenshot_20240203_020542_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/f6cb![Screenshot_20240203_020550_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/b5d2f1ce-3273-4745-bb73-a96afdf785d5)

t_20240123_234946_File Viewer.jpg …]()

![Uploading Screenshot_20240203_020550_Chrome.jpg …]()

![Screenshot_20240123_234939_File Viewer.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/827dac64-bf28-4519-8392-72799fb04449)

![Screenshot_20240123_235120_File Viewer.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/67f3b971-2dbf-44c2-87a0-3c454ee92a33)

![Screenshot_20240123_234930_File Viewer.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/3b643f32-2418-4502-8c6e-c![Screenshot_20240123_091932_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/7e9597db-104b-49b7-b4ed-f0e51b2c4f1f)

e Viewer.jpg …]()

![![Screenshot_20240118_031105_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/8088ab47-05fd-46c9-9b9a-5448a0b6907f)

)

![Screenshot_20240123_091945_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/92e103d9-ddaf-4d47-9596-1646bf36b9e8)

![Uploading Screensho![Screenshot_20240120_152609_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/441d4c16-859f-4e82-aea7-9bc7446a2898)

t_20240123_234851_File Viewer.jpg …]()

![Uploading Screenshot_20240120_152609_Chrome.jpg …]()

![Uploading Screenshot_20240118_031105_Chrome.jpg …]()

<!-- Error while uploading Screenshot_20240117_090706_Gallery.jpg -->![Screenshot_20240115_212255_Samsung Internet.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/4c0db99f-d95f-421a-b7d6-f75042862992)

![Screenshot_20240105_045051_Bitcoincom Wallet.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/9cfb99d8-f674-4cec-a5a3-873041d9d28b)

![Screenshot_20240115_212304_Samsung Internet.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/6f9046d0-e7bf-4278-ae63-31c9ffd4da05)

![Screenshot_20240103_042431.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/26aa4578-d3c5-42d9-bbfa-1c68f75a42d0)

<!-- Error while uploading Screenshot_20240104_052817_One UI Home.jpg -->![Screenshot_20240102_183300_OfficeSuite.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/211ecf3f-6f15-4b77-9493-c436c1a266ab)

![Screenshot_20240101_140639_Instabridge.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/24dab677-ca36-4f69-8f75-1bbc74f43409)

![Screenshot_20240101_140516_Instabridge.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/00c171fa-4c95-46ae-9692-8b41d094d316)

![Screenshot_20240101_140509_Instabridge.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/b14531a0-03f8-4b1b-87a9-0994f139fda6)

![Screenshot_20240101_140526_Instabridge.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/f9e18778-b5b2-4fa2-b755-498b96682021)

![Screenshot_20240101_140442_Instabridge.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/4edefdaa-3889-43ca-a72d-8c5c049765a2)

![Screenshot_20240101_140429_Instabridge.jpg](https://github.![Screenshot_20240101_140449_Instabridge.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/1090782c-3c4c-44cb-abf5-117125e27f0e)

7578-6262-4c77-b149-81e314fef2a7)

![Uploading Screenshot_20240101_140449_Instabridge.jpg …]()

![Screenshot_20240101_140458_Instabridge.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/2edf952c-86fa-48d9-9c45-0a31064d5760)

![Screenshot_20240101_140436_Instabridge.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/c9080427-0ad6-4a48-972d-a6e0ac66b443)

![Screenshot_20231215_224133_Instabridge.jpg](https://github.![Screenshot_20231215_223954_Instabridge.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/0014eb7c-0e57-41f7-88c4-3fa7c371be83)

cce3-7920-472a-a53d-20ef7cb7e1aa)

![Uploading Screenshot_20231215_223954_Instabridge.jpg …]()

<!-- Error while uploading Screenshot_20231231_025328_Drive.jpg -->![Screenshot_20231215_223947_Instabridge.jpg](https://github.com/coinbase/paymaster-bun![Screenshot_20231215_223905_Instabridge.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/d046b6fd-37f6-4418-9211-f7e396d36675)

a1759c)

![Screenshot_20231215_223940_Instabridge.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/7f6e2352-f626-42b4-8bba-6343f74e0f38)

![Screenshot_20231215_223925_Instabridge.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/d51cae3e-6d38-4909-955f-0841d0c5c580)

![Uploading Screenshot_20231215_223905_Instabridge.jpg …]()

![Screenshot_20231215_223746_Instabridge.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/7b141eab-9c43-424c-a432-f63634360514)

![Screenshot_20231215_223853_Instabridge.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/31a904ad-5301-43a5-9d7e-2674353b2ce5)

![Screenshot_20231215_223725_Instabridge.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/89503343-afe5-46d4-acb0-c15683db692b)

![Screenshot_20231215_223637_Instabridge.jpg](https://github.![Screenshot_20231215_223621_Instabridge.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/fe415a2e-8dcd-4b20-b177-f9194a495373)

2364-8a24-4d77-a7b3-8f2c7bcab99d)

![Uploading Screenshot_20231215_223621_Instabridge.jpg …]()

![Uploading Sc![Screenshot_20231214_224909_Instabridge.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/d7d05cda-ce37-4dcf-9559-d96a533faf1f)

16_Instabridge.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/6053d38c-a319-4dab-a488-7ca9626270ac)

![Uploading Screenshot_20231214_224909_Instabridge.jpg …]()

![Screenshot_20231213_060053_Drive.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/dffbb2ed-7f9d-4e85-8e48-d85e41d2d193)

![Screenshot_20231204_231349_Phone.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/f2d64841-e7a2-49b7-8302-048d1fa67fbe)

![Screenshot_20231206_174540_Uniswap.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/63d48654-fac8-45ac-8e18-353ae4aaa181)

![Screenshot_20231130_215655_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/3383983c-f08a-4362-8f10-e487f6adfc8c)

![Screenshot_20231209_122742_Phone.jpg](https://github.![Screenshot_20231215_223733_Instabridge.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/e0f6477e-e52e-4670-9056-35e61e98dfb4)

ab5b-7c02-4660-9a88-9dd20025e6bd)

![Uploading Screenshot_20231215_223733_Instabridge.jpg …]()

![Screenshot_20231215_223846_Instabridge.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/1b256063-5e18-4777-8af9-680c660d02d4)

![Screenshot_20231215_223932_Instabridge.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/3618f276-bbe0-4dd6-9859-082cd179f551)

![Screenshot_20231219_112409_Instabridge.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/e4acca50-c2c9-46ce-8a80-bec1c1da6c24)

![Screenshot_20231219_112426_Instabridge.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/f35ff621-f957-4192-abe4-1f2ca076341e)

![Uploading Sc![Screenshot_20231219_124746.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/4012944d-79d7-49d6-af50-936864f6e7e3)

Uploading Screenshot_20231219_115704.jpg …]()

![Uploading Screenshot_20231219_124746.jpg …]()

![Screenshot_20231227_155006.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/bee89fe5-67f8-4e4e-b200-a7c1051e908e)

![Screenshot_20231220_173045_Instabridge.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/d0ef994f-f82c-4fcc-aa9c-6f420dcd1c88)

<!-- Error while uploading Screenshot_20231227_155101.jpg --><!-- Error while uploading Screenshot_20231227_155116.jpg --><!-- Error while uploading Screenshot_20231227_155111.jpg --><!-- Error while uploading Screenshot_20231227_155106.jpg --><!-- Error while uploading Screenshot_20231227_155122.jpg --><!-- Error while uploading Screenshot_20231227_155129.jpg --><!-- Error while uploading Screenshot_20231227_155143.jpg -->![Screenshot_20231229_164042_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/36e24756-7ff9-4ab6-b31c-c97c125dfb3e)

![Screenshot_20231231_025538.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/db716642-fbc4-4486-a25c-a9bbd4816aba)

![Screenshot_20231228_091732_Cash App.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/b316289c-a087-49e8-9a4e-5f5b2d8e8f26)

![Screenshot_20240101_140537_Instabridge.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/f48912f5-c4b4-43f4-bfdd-b972e5d6d04c)

<!-- Error while uploading Screenshot_20231227_155136.jpg --><!-- Error while uploading Screenshot_20240111_221100_Maps.jpg -->![Screenshot_20240106_211340_Outlook.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/9e331c84-60aa-455f-8337-cc8001389e65)

![Uploading Screens![Screenshot_20240131_184253_CoinStats.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/1a8d924c-fd13-48c0-a987-4bf74e71aa6c)

_20240130_030000_CoinStats.jpg …]()

![Uploading Screenshot_20240131_184253_CoinStats.jpg …]()

![Screenshot_20240130_211252_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/0b79cd9b-d516-4071-a17c-20f85de4e5f9)

![Screenshot_20240203_015244_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/3fc5da51-d5c4-4335-a31e-65b097bb423e)

![Screenshot_20240203_015254_Chrome.jpg](https://github.![Screenshot_20240203_015307_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/b5c9d0dc-76ae-4fed-858a-2fe9a45d8214)

/27996b78-80bc-4a46-8642-9565a3def0af)

![Uploading Screenshot_20240203_015307_Chrome.jpg …]()

![Screenshot_20240203_015316_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/4d30723f-ab34-426a-935f-cca6ee4cc8a7)

![Screenshot_20240203_015357_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/9a676174-37aa-4188-a731-9787e4911f46)

![Screenshot_20240203_015333_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/e31b5102-b2d1-44b0-95dd-15ac06cf6fc7)

![Screenshot_20240203_015404_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/1331d6ff-8c67-4ce2-a933-163929192b30)

![Screenshot_20240203_015444_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/7c99c881-32c7-47d6-abc6-50a9ba0b1326)

![Screenshot_20240203_015430_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/d861193b-fa1d-4c28-9c4e-1cf0f10bd887)

![Screenshot_20240203_015458_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242![Screenshot_20240203_015521_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/3f2e31f7-a8e6-4478-8f2e-eb186b8e1baf)

enshot_20240203_015556_Chrome.jpg …]()

![Uploading Screenshot_20240203_015521_Chrome.jpg …]()

![Screenshot_20240203_015631_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/4de6b9fd-bf08-4fb9-a96e-85583aa9b0ab)

![Screenshot_20240203_015650_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/4c288b71-3cda-4c3b-a67f-74b19c1429ef)

![Screenshot_20240203_015640_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/eb164dae-83a1-4f4c-be95-3b811f7b2b4e)

![Screenshot_20240203_015752_Chrome.jpg](https://github.![Screenshot_20240203_015654_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/253597ae-59b3-48b1-8387-5e47d11f8e6c)

/407a2c33-41e5-4b4d-b526-43544f0a510c)

![Uploading Screenshot_20240203_015654_Chrome.jpg …]()

![Screenshot_20240203_015726_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/32f7d976-2a0d-4a68-ad14-d23a2fd6c6c0)

![Screenshot_20240203_015736_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/471d0705-822d-441a-b46e-7adaad6cb456)

![Screenshot_20240203_015822_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/c65d933e-99f6-455d-b0f8-952ba16d6017)

![Screenshot_20240203_015810_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/4e6e6a89-4fc3-4925-b01c-f2365b289943)

![Screenshot_20240203_015841_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/d98544e8-8bff-4209-bb34-d3c465067526)

![Screenshot_20240203_015848_Chrome.jpg](https://github.com/coinbase/pay![Screenshot_20240203_015948_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/afd66b23-a836-4d57-b986-e1102a3385a9)

49f-a944-5d8298e86ba2)

![Screenshot_20240203_015911_Chrome.jpg](https://github.![Screenshot_20240203_015928_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/c9287061-0613-4709-9d55-1e7d0da137e0)

/88e72abb-b038-4214-9cac-4b2c1752bffa)

![Uploading Screenshot_20240203_015928_Chrome.jpg …]()

![Uploading Screenshot_20240203_015948_Chrome.jpg …]()

![Screenshot_20240203_015918_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/40c2ddee-8aac-432c-94c3-1d0c345c17d9)

![Screenshot_20240203_020000_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/479660e3-1767-47d3-919a-cbc9d42996a5)

![Screenshot_20240203_020008_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/5f677080-96fb-48d0-ae44-465054a41e4c)

![Screenshot_20240203_015955_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/933ff472-e26a-4c8a-b9aa-ceee6dd81977)

![Screenshot_20240203_020053_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/5d25ba23-023e-453b-8f9b-7055b8f9a186)

![Screenshot_20240203_020044_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/9754025a-52cd-4494-8b54-5ce8d0640802)

![Screenshot_20240203_020034_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/3795087f-0fe5-4d71-a49d-345fd165439c)

![Screenshot_20240203_020131_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/8eec750a-0dc1-43e7-8911-da6694c96f26)

![Screenshot_20240203_020100_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/232cb164-41bf-40a3-92ec-6f43bb8357b1)

![Screenshot_20240203_020022_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/8919916b-5854-4fcc-8096-4c3ca3533b68)

![Screenshot_20240203_020201_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/13214d86-98a5-4197-be5c-a7f203d99e6c)

![Screenshot_20240203_020114_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/797260c2-c8ac-48b4-a7bb-6bfa4da6f1e0)

![Screenshot_20240203_020223_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/5ab1f7c8-0a9b-4f5b-a9ed-e389875067b6)

![Screenshot_20240203_020152_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/a28114c5-9b25-4185-83ce-2695dc3bcd55)

![Uploading Scre![Screenshot_20240203_020308_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/4e44997a-d8c5-407f-aaed-adf10751d79d)

enshot_20240203_020249_Chrome.jpg …]()

![Uploading Screenshot_20240203_020308_Chrome.jpg …]()

![Screenshot_20240203_020233_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/19946fa7-43e8-430b-bcaa-9efeee1181e4)

![Screenshot_20240203_020318_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/a02446e1-dbb1-48e4-b614-b8cd33f689b9)

![Uploading Scre![Screenshot_20240203_020347_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/0e1fb75b-7a04-4363-b2d2-3c1a1a8226e8)

![Screenshot_20240203_020400_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/e5fd0faa-0c30-47bd-9f6a-6f4632169595)

enshot_20240203_020347_Chrome.jpg …]()

![Uploading Screenshot_20240203_020400_Chrome.jpg …]()

![Uploading Scre![Screenshot_20240203_020429_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/5e87e4c8-d8b7-496b-b28c-9a131a40aa3d)

40203_020352_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242![Screenshot_20240203_020435_Chrome.jpg](https://github.![Screenshot_20240203_020444_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/221cf9a5-c95c-4299-a619-231e7dcf475e)

/1eeb4971-4dd3-431e-af87-a161b2abe508)

enshot_20240203_020429_Chrome.jpg …]()

![Uploading Screenshot_20240203_020435_Chrome.jpg …]()

![Uploading Screenshot_20240203_020444_Chrome.jpg …]()

![Screenshot_20240203_020454_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/a15694f0-2ffd-4261-9cf2-f060bd498070)

![Uploading Screenshot_20240203_![Screenshot_20240203_020603_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/943a1ab9-a174-4e1f-98c4-860e433865a4)

020521_Chrome.jpg …]()

![Screenshot_20240203_020529_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/b59e56a4-23ff-4f69-8e80-cf1c7e6024a7)

![Screenshot_20240203_020619_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/da600ee7-62f8-4731-b1e0-72d54d485806)

![Uploading Scre![Screenshot_20240203_020628_Chrome.jpg](https://github.![Screenshot_20240203_020650_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/4a4814bf-5b2f-4632-9487-2633232ae4ed)

![Screenshot_20240203_020636_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/3360640c-40d2-4138-8045-9988df8980de)

020610_Chrome.jpg …]()

![Uploading Screenshot_20240203_020628_Chrome.jpg …]()

![Uploading Screenshot_20240203_020650_Chrome.jpg …]()

![Uploading Screenshot_20240203_020636_Chrome.jpg …]()

![Screenshot_20240203_020703_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/ca85dd15-3e9a-4497-8e09-4a2dc7e5bc08)

![Uploading Scre![Screenshot_20240203_020716_Chrome.jpg](https://github.com/coinbase/paymaster-bundler-examples/assets/153387242/29b54312-6ed2-4cb7-9f2c-4ebca33e1ddb)

enshot_20240203_020656_Chrome.jpg …]()

![Uploading Screenshot_20240203_020716_Chrome.jpg …]()

Reply:
That's just a beginning.Should I go on and I haven't received no one sent from the stuff.Not one penny i've already received

Reply:
Nothing have I received from this and I got plenty dirt left to go if you want me to send you some more.I'll send it all to you.I got everything he's a dummy wrong.I need other people.He's done wrong as well.Since nineteen ninety four